content/zh/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -284,7 +307,7 @@ the monitoring system.
 
 Pod 的资源使用情况被报告为 Pod 状态的一部分。
 
-如果为集群配置了 [可选监控](/docs/tasks/debug-application-cluster/resource-metrics-pipeline/)，则可以从监控系统检索 Pod 资源的使用情况。
+如果为集群配置了 [可选监控](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/cluster-monitoring/README.md)，则可以从监控系统检索 Pod 资源的使用情况。
```

Comment:
@gaoguangze111 
Thanks for your job!
pls deal with the conflicts and squash your commits.